<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-10</y>
      <width>771</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="frame_superior">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_contenido">
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(144, 116, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>701</width>
          <height>471</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QWidget" name="">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>20</y>
              <width>311</width>
              <height>431</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QListWidget" name="lista_musica">
               <property name="styleSheet">
                <string notr="true">color: rgb(144, 116, 255);</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="spacing">
                <number>15</number>
               </property>
               <property name="bottomMargin">
                <number>15</number>
               </property>
               <item>
                <widget class="QLabel" name="titulo_cancion">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(144, 116, 255);</string>
                 </property>
                 <property name="text">
                  <string>TITULO:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="titulo_artista">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(144, 116, 255);</string>
                 </property>
                 <property name="text">
                  <string>ARTISTA:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="titulo_album">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(144, 116, 255);</string>
                 </property>
                 <property name="text">
                  <string>ALBUM:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="numero_cancion">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(144, 116, 255);</string>
                 </property>
                 <property name="text">
                  <string>NUMERO:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QFrame" name="frame">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 0, 0);</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QWidget" name="">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>10</y>
                <width>311</width>
                <height>201</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QPushButton" name="bt_abrir">
                 <property name="font">
                  <font>
                   <family>OCR A Extended</family>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(144, 116, 255);</string>
                 </property>
                 <property name="text">
                  <string>Abrir Musica</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../botones/Abrir_Musica.png</normaloff>../botones/Abrir_Musica.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="grafica_layout"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 0, 0);
</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QWidget" name="">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>321</width>
                <height>211</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QSlider" name="slider_tiempo">
                 <property name="styleSheet">
                  <string notr="true">QSlider::groove:horizontal{
	border: 4px solid rgb(144, 116, 255);
	height: 4px;
	background:rgb(144, 116, 255);
}

QSlider::handle:horizontal{
	background:rgb(144, 116, 255);
	margin: -4px;
}

QSlider::add-page:horizontal{
background-color:black;
border:2px solid rgb(144, 116, 255);
border-radius: 3px
}

</string>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="indicador_tiempo">
                   <property name="styleSheet">
                    <string notr="true">color: rgb(144, 116, 255);
font: 12pt &quot;Magnum&quot;;</string>
                   </property>
                   <property name="text">
                    <string>00:00</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QPushButton" name="bt_anterior">
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(144, 116, 255);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>../botones/anterior.png</normaloff>../botones/anterior.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bt_play">
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(144, 116, 255);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>../botones/play.png</normaloff>../botones/play.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bt_pausa">
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(144, 116, 255);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>../botones/pausa.png</normaloff>../botones/pausa.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bt_siguiente">
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(144, 116, 255);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>../botones/siguiente.png</normaloff>../botones/siguiente.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pb_aleatorio">
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(144, 116, 255);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>../botones/aleatorio.png</normaloff>../botones/aleatorio.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QPushButton" name="bt_volumen_0">
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(144, 116, 255);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>../botones/mute.png</normaloff>../botones/mute.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bt_volumen_50">
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(144, 116, 255);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>../botones/bajo.png</normaloff>../botones/bajo.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bt_volumen_100">
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(144, 116, 255);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>../botones/alto.png</normaloff>../botones/alto.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="slider_volumen">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QSlider::groove:horizontal{
	border: 1px solid rgb(144, 116, 255);
	height: 4px;
	background:rgb(144, 116, 255);
}

QSlider::handle:horizontal{
	background:rgb(144, 116, 255);

	width: 28px;
	height: 28px;

left: 11px;
rigth: 11px;

	margin: -12px;
	border-radius: 14px;
}

QSlider::add-page:horizontal{
background-color:white;
border:1px solid white;
}



</string>
                   </property>
                   <property name="maximum">
                    <number>100</number>
                   </property>
                   <property name="value">
                    <number>50</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="indicador_volumen">
                   <property name="styleSheet">
                    <string notr="true">color: rgb(144, 116, 255);
font: 12pt &quot;Magnum&quot;;</string>
                   </property>
                   <property name="text">
                    <string>50</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
